__tests__/app.test.js

```diff
@@ -21,3 +21,35 @@ describe("/api/topics", () => {
             });
     });
 });
+
+describe("/api", () => {
+    test("GET:200 responds with an object describing all available endpoints for this API. The object includes a description, a queries array and an example response", () => {
+        return request(app)
+            .get("/api/")
+            .expect(200)
+            .then(({ body: enpointsObject }) => {
+                const validPaths = [
+                    "GET /api",
+                    "GET /api/topics",
+                    "GET /api/articles",
+                    // TODO update as routes are defined
+                ];
```

Comment:
Rather than updating this each time we add an endpoint, we can make it flexible by simply expecting the response to equal the .json file. If you import the .json file into your tests, you'll be able to do this. Should also mean you don't need to do the rest of the logic underneath for checking if the endpoint is there. 👍

Reply:
Ahh yes, good shout. I didn't think of this. I will re-factor and submit a new PR.